marketing/styles.css

```
@@ -0,0 +1,331 @@
+  --line: #25314d;
+  --ok: #34d399;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html,
+body {
+  margin: 0;
+  font-family: "Inter", system-ui, -apple-system, sans-serif;
```

Stylelint (17.5.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)